### PR TITLE
fix: Change padding in modern UiCard component according to the newest design

### PR DESCRIPTION
**Pull request by jbucior**
image
image

### CODE REVIEW

**Review by AlanLes — DISMISSED** [review]
LGTM 👍 📘 😄

**Review by sonya0504 — APPROVED** [review]

File: src/components/organisms/UiCard/UiCard.vue [hunk 1]
```diff
@@ -247,8 +247,11 @@ const defaultProps = computed(() => ({
   &--modern {
     @include mixins.override-logical(container, $element, padding, 0);
 
+    gap: functions.var($element + "-modern", gap, 0);
```
Comment (pspaczek) [inline src/components/organisms/UiCard/UiCard.vue:250]:
```suggestion
    gap: functions.var($element, gap, 0);
```

File: src/components/organisms/UiCard/UiCard.vue [hunk 2]
```diff
@@ -247,8 +247,11 @@ const defaultProps = computed(() => ({
...
+
     @include mixins.from-tablet {
       flex-direction: row-reverse;
+      gap: functions.var($element + "-modern-tablet", gap, 0);
```
Comment (pspaczek) [inline src/components/organisms/UiCard/UiCard.vue:254]:
```suggestion
```
UiCard has 
```
 @include mixins.from-tablet {
    flex-direction: row;
    gap: functions.var($element + "-tablet", gap, 0);
  }
```

**Review by pspaczek — CHANGES_REQUESTED** [review]
Need a few changes

**Pull request merged**